getting-started/README.md

```diff
@@ -1,110 +1,108 @@
+8. `ssh` into the instance using the floating IP (`ssh -i <key> ubuntu@<floating-ip>`)
 
+
+
+## Deploy the ChatQnA Solution
+Use the command below to install docker on a clean virtual machine
+```bash
+wget https://raw.githubusercontent.com/opea-project/GenAIExamples/refs/heads/main/ChatQnA/docker_compose/install_docker.sh
+chmod +x install_docker.sh
+./install_docker.sh
 ```
+Configure Docker to run as a non-root user by following these [instructions](https://docs.docker.com/engine/install/linux-postinstall/)
+
+Clone the repo and navigate to ChatQnA
+
+```bash
 git clone https://github.com/opea-project/GenAIExamples.git
 cd GenAIExamples/ChatQnA
 ```
-### Set the required environment variables:
-```
-# Example: host_ip="192.168.1.1"
-export host_ip="External_Public_IP"
-# Example: no_proxy="localhost, 127.0.0.1, 192.168.1.1"
-export no_proxy="Your_No_Proxy"
+Set the required environment variables:
+```bash
+export host_ip="localhost"
 export HUGGINGFACEHUB_API_TOKEN="Your_Huggingface_API_Token"
 ```
-If you are in a proxy environment, also set the proxy-related environment variables:
-```
-export http_proxy="Your_HTTP_Proxy"
-export https_proxy="Your_HTTPs_Proxy"
-```
-
-Set up other specific use-case environment variables by choosing one of these options, according to your hardware:
 
+Set up other specific use-case environment variables:
+```bash
+cd docker_compose/intel/cpu/xeon/
+source set_env.sh
 ```
-# on Xeon
-source ./docker_compose/intel/cpu/xeon/set_env.sh
-# on Gaudi
-source ./docker_compose/intel/hpu/gaudi/set_env.sh
-# on Nvidia GPU
-source ./docker_compose/nvidia/gpu/set_env.sh
+Now we can start the services
+```bash
+docker compose up -d
 ```
+It takes a few minutes for the services to start. Check the logs for the services to ensure that ChatQnA is running.
+For example to check the logs for the `tgi-service`:
 
-## Deploy ChatQnA Megaservice and Microservices
-Select the directory containing the `compose.yaml` file that matches your hardware.
+```bash
+docker compose logs tgi-service | grep Connected
 ```
-#xeon
-cd docker_compose/intel/cpu/xeon/
-#gaudi
-cd docker_compose/intel/hpu/gaudi/
-#nvidia
-cd docker_compose/nvidia/gpu/
+The output shows `Connected` as shown:
 ```
-Now we can start the services
+tgi-service | 2024-10-18T22:41:18.973042Z INFO text_generation_router::server: router/src/server.rs:2311: Connected
 ```
-docker compose up -d
-```
-It will automatically download the needed docker images from docker hub:
 
-- docker pull opea/chatqna:latest
-- docker pull opea/chatqna-ui:latest
+### Interact with ChatQnA
 
-In the following cases, you will need to build the docker image from source by yourself.
+You can interact with ChatQnA via a browser interface:
+* Under `Infrastructure` in the left pane, go to `Network/Security groups/<Your Security Group>/Rules`
+* Select `Create`
+* Enable inbound traffic for port 80
+* Open a browser and navigate `http://{external_public_ip}:80` to view the ChatQnA interface.

```

Comment:
recommend making this more explicit by modifying this step to be: 

* To view the ChatQnA interface, open a browser and navigate to the UI by inserting your externally facing IP address in the following:  `http://{external_public_ip}:80'.